pdr_backend/analytics/predictoor_dashboard/dash_components/plots.py

```diff
@@ -0,0 +1,151 @@
+from typing import Union, List, Tuple, Optional
+import plotly.graph_objects as go
+from enforce_typing import enforce_types
+from pdr_backend.util.time_types import UnixTimeS
+
+
+@enforce_types
+def create_scatter(
+    name: str, x_data: List[float], y_data: List[int], mode="lines"
+) -> go.Scatter:
+    """Create a scatter plot with the given data.
+    Args:
+        name (str): Name of the trace.
+        x_data (List): List of x values.
+        y_data (List): List of y values.
+        mode (str): Plot mode. Default is 'lines'.
+    Returns:
+        go.Scatter: Scatter plot trace.
+    """
+    return go.Scatter(x=x_data, y=y_data, mode=mode, name=name)
```

Comment:
this looks like it could just be a ** call. Why create extra functions?
i.e. go.Scatter(**{x=x_data, ...})